api/v1beta1/types.go

```diff
@@ -64,6 +64,30 @@ type NetworkDetails struct {
 	AssignPrivateDnsRecord *bool `json:"assignPrivateDnsRecord,omitempty"`
 }
 
+type VnicAttachment struct {
+	// VnicAttachmentId defines the ID of the VnicAttachment
+	VnicAttachmentId *string `json:"vnicAttachmentId,omitempty"`
+
+	// AssignPublicIp defines whether the instance should have a public IP address
```

Comment:
whether the instance -> whether the vnic?

Reply:
fixed